source/src/BNFC/Backend/Java/CFtoAntlr4Parser.hs

```diff
@@ -53,8 +53,9 @@ type MetaVar     = (String, Cat)
 
 -- | Creates the ANTLR parser grammar for this CF.
 --The environment comes from CFtoAntlr4Lexer
-cf2AntlrParse :: String -> String -> CF -> SymEnv -> String
-cf2AntlrParse packageBase packageAbsyn cf env = unlines
+--The bool is line numbering, which is currently not supported.
```

Comment:
Put argument documentations down as haddock comments, i.e, a `-- ^` comment following the argument type.  See https://github.com/andreasabel/haskell-style-guide/blob/master/haskell-style.md